akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -845,6 +847,114 @@ trait FlowOps[+Out, +Mat] {
    */
   def mapAsyncUnordered[T](parallelism: Int)(f: Out ⇒ Future[T]): Repr[T] = via(MapAsyncUnordered(parallelism, f))
 
+  /**
+   * Use the `ask` pattern to send a request-reply message to the target `ref` actor.
+   * If any of the asks times out it will fail the stream with a [[akka.pattern.AskTimeoutException]].
+   *
+   * Do not forget to include the expected response type in the method call, like so:
+   *
+   * '''
```

Comment:
should those be backticks?

Reply:
or `{{{`/`}}}`?

Reply:
''' actually works AFAIR, but can change to {{{